octopoes/octopoes/models/ooi/findings.py

```diff
@@ -1,13 +1,30 @@
+import abc
+from enum import Enum
 from typing import Literal, Optional
 
+from pydantic import AnyUrl
+
 from octopoes.models import OOI, Reference
 from octopoes.models.persistence import ReferenceField
 
 
-# todo: make abstract
-class FindingType(OOI):
+class RiskLevelSeverity(Enum):
+    CRITICAL = "critical"
+    HIGH = "high"
+    MEDIUM = "medium"
+    LOW = "low"
+    RECOMMENDATION = "recommendation"
+
+
+class FindingType(OOI, abc.ABC):
```

Comment:
Why inherit from `abc.ABC`? This on itself doesn't really add anything. `abc.ABC` has very specific functionality that we don't seem to be using, so I don't really see the point of inheriting from `abc.ABC`.

Reply:
@dekkers 

`FindingType` is an abstract class, so it should never be instantiated directly. So, IMO better to enforce it?

Reply:
But that is not what `abc.ABC` does. See for example:

```
In [1]: import abc

In [2]: class Test(abc.ABC):
   ...:     pass
   ...: 

In [3]: a = Test()

In [4]: print(a)
<__main__.Test object at 0x7f5423fba4d0>
```

Reply:
Good points. Technically it doesn't add anything, semantically it has meaning. I don't really have a strong opinion on this, but I'd suggest leaving it out as it's not supposed to be used like this.

Reply:
Fixed